lib/Conversation/styles.scss

```diff
@@ -1,9 +1,16 @@
 /* ==========================================================================
    Config
    ========================================================================== */
+$conversation-active-background-color: #ffffff !default;
+$conversation-active-shadow-color: rgba(158, 158, 158, 0.17) !default;
+$conversation-active-shadow-size: 0px 0px 5px 0px !default;
+
+
 $conversation-border-radius: 5px !default;
 $conversation-padding: $layout-spacing-base !default;
 
+$conversation-margin: 10px 0 !default;
```

Comment:
For spacing (or any context) if a value is derived from a design pattern then we could use the base setting to create our desired output.

In simple terms, spacing is based on a `20px` setting called `$layout-spacing-base`. To get a 10px spacing you can do `$layout-spacing-base/2` and then that value is derived from one of our base settings. Making it more consistent etc...